common/oauthTokenManager.go

```diff
@@ -118,13 +112,9 @@ func (uotm *UserOAuthTokenManager) GetTokenInfo(ctx context.Context) (*OAuthToke
 		if err != nil { // this is the case when env var exists while get token info failed
 			return nil, err
 		}
-	} else { // Scenario: session mode which get token from cache
```

Comment:
What scenarios is this removing?

Reply:
When we try to figure out what type of credential to use, AzCopy makes no attempt to poll the cache on this branch, and falls back to "Unknown", which should always generate an error on Blob.

Reply:
Hey @adreed-msft ,
Could you please clarify the specific scenarios you're encountering? Both logging in and copying data seem to be functioning correctly on my end. Initially, I log in via DEVICE-Login, then proceed to copy data between two blobs, which operates smoothly. Even after logging out successfully, I repeat the process by logging in again, and the scenario continues to work seamlessly.

Reply:
Have you tried performing the login status command?